apps/services/user-notification/src/buildOpenApi.ts

```diff
@@ -6,4 +6,5 @@ buildOpenApi({
   path: 'apps/services/user-notification/src/openapi.yaml',
   appModule: AppModule,
   openApi,
+  enableVersioning: true,
```

Comment:
pósthólf is not ready for this change to the best of my knowledge - so adding "v1" to the endpoints needs to be coordinated with the pósthólf team

Reply:
The api is already using versioning on the routes and the pósthólf is already making request to `/v1` endpoint. This is to make the openapi document describe correct routes, as the version is missing.

I have synced with Björn Breki about their current integration which confirmed that this change is OK.